projects/canopy-test-app/src/test.ts

```diff
@@ -22,6 +22,9 @@ declare const require: {
 getTestBed().initTestEnvironment(
   BrowserDynamicTestingModule,
   platformBrowserDynamicTesting(),
+  {
+    teardown: { destroyAfterEach: false },
```

Comment:
Does using the default `true` behaviour break a load of stuff?

Reply:
@elenagarrone If we're using true, then we can remove these lines altogether since it's the default in ng14